docs/api/instrument/index.rst

```diff
@@ -0,0 +1,25 @@
+.. _instrument_api :
+
+qcodes.instrument (Instruments)
+===============================
+
+.. autosummary::
+
+    qcodes.instrument
+    qcodes.instrument.visa
+    qcodes.instrument.ip
```

Comment:
"creating instrument drivers" says
> - `IPInstrument` - a deprecated driver just for ethernet connections. Do not use this; use `VisaInstrument` instead.

Shall we not include it in the docs then?